common/views/components/RelevanceRater/RelevanceRater.js

```diff
@@ -45,99 +26,27 @@ const RelevanceRater = ({
   workType,
   _queryType,
 }: Props) => {
+  const [showRatings, setShowRatings] = useState(false);
+  useEffect(() => {
+    setShowRatings(true);
```

Comment:
Should we go with the pattern `isEnhanced`?

Reply:
I think you're right that's clearer. Will change (although I think we have shouldRender and maybe some others, elsewhere)